src/components/ebay-breadcrumb/index.js

```diff
@@ -0,0 +1,43 @@
+const markoWidgets = require('marko-widgets');
+const emitAndFire = require('../../common/emit-and-fire');
+const processHtmlAttributes = require('../../common/html-attributes');
+const template = require('./template.marko');
+
+function getTemplateData(state, input) {
+    const htmlAttributes = processHtmlAttributes(input);
+    const inputItems = input.items || [];
+    let items = (inputItems).map((item, index) => {
+        const itemHtmlAttributes = processHtmlAttributes(item);
+        let tag = 'a';
+        let ariaCurrent = null;
+        const href = item.href || null;
+        const current = ((inputItems.length - 1) === index);
+        if (current && !href) {
+            tag = 'span';
+            ariaCurrent = 'page';
+        }
+        return {
+            tag,
+            htmlAttributes: itemHtmlAttributes,
+            renderBody: item.renderBody,
+            href,
+            ariaCurrent
+        };
+    });
+    items = Object.keys(items).length > 0 ? items : null;
+    return {
+        items,
+        headingText: input.headingText || '',
+        headingLevel: input.headingLevel || 'h2',
+        htmlAttributes: htmlAttributes
+    };
+}
+function handleClick(event, el) {
```

Comment:
@mlrawlings @DylanPiercey What does Marko pass in to the second parameter `el`? Is it `event.target` or `event.currentTarget`?

Reply:
Emitting 3 values now - { event, target: event.target, currentTarget: el } since `event.currentTarget` is pointing to body

Reply:
This doesn't feel consistent with the event fired by menu component. Let's meet to discuss.